main.html

```diff
@@ -0,0 +1,365 @@
+          <div>
+            <h5>Photoshop</h5>
+            <p>86, 356, 213 students</p>
+          </div>
+          <div>
+            <h5>Grapics Design</h5>
+            <p>101, 456, 789 students</p>
+          </div>
+          <div>
+            <h5>Drawing</h5>
+            <p>42, 896, 234 students</p>
+          </div>
+        </summary>
+        <summary class="feature_article">
+          <h4>Business</h4>
+          <div>
+            <h5>Financial Analysis</h5>
+            <p>123, 456, 789 students</p>
+          </div>
+          <div>
+            <h5>SQL</h5>
+            <p>36, 390, 213 students</p>
+          </div>
+          <div>
+            <h5>PMP</h5>
+            <p>37, 395, 273 students</p>
+          </div>
+        </summary>
+      </article>
+    </section>
+
+    <!-- Footer Section -->
+    <footer>
+      <div class="partnership">
+        <div class="partnership_heading">
+          <h2>Partner</h2>
+          <div class="partnership_underline"></div>
+        </div>
+        <img
+          src="./assets/-31600822121rk0lmm4sqe.png"
+          alt="airbnb logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/apple-logo.png"
+          alt="apple logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/google-logo.png"
+          alt="google logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/mozilla-logo-11609369622n1zxmrjknf.png"
+          alt="mozilla logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/microsoft-logoo.jpg"
+          alt="microsoft logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+      </div>
+
+      <ul class="footer_lists">
+        <li class="footer_list">
+          <a href="#" class="footer_list_a"
+            ><i class="fa fa-twitter" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li class="footer_list">
+          <a href="#" class="footer_list_a"
+            ><i class="fa fa-facebook" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li class="footer_list">
+          <a href="#" class="footer_list_a"
+            ><i class="fa fa-github" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li class="footer_list">
+          <a href="#" class="footer_list_a"
+            ><i class="fa fa-linkedin" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li class="footer_list">
+          <a href="#" class="footer_list_a"
+            ><i class="fa fa-whatsapp" aria-hidden="true"></i
+          ></a>
+        </li>
+      </ul>
+      <p class="footer_paragraph">&copy; 2022 Fayob inc. All right reserved</p>
+    </footer>
```

Comment:
- [ ] Kindly make a footer like one in the design

 | Your design  | Behance design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/190259254-1e5b3ebe-d8d5-4d9c-a3cd-e489e3b01dec.png)  | ![image](https://user-images.githubusercontent.com/53402600/190259348-e4b3205b-1bfb-4693-af04-495f63c9b638.png)|